content/contributing/membership/bronze/contents.lr

```diff
@@ -0,0 +1,41 @@
+name: Bronze
+---
+cost: 200
+---
+title: Bronze
+---
+showcard: yes
+---
+body:
+
+Bronze members can have their name listed on the `Members page </community/members/>`__, along with a medium sized logo, name, and link to a company webpage of their choice.
+
+If the member company wishes to advertise their `consulting </community/consulting/>`__, `training </community/training/>`__ or `software development </community/development/>`__ services, they will receive a standard listing on the services pages of their choice.
+
+If you are an independent consultancy, sole proprietorship, or other small business, we invite you to become a **Bronze Member** of the BeeWware community.
+
+---
+display: yes
+---
+color: bronze
+---
+description:
+
+* Vote on project priorities
+* Listed on Members page:
+    * Name
+    * Medium logo
+    * Link to company website
+    * *No brief company description*
+* Standard placement on services pages
+* *No logo in rotation on homepage*
```

Comment:
I'd rather see these "not included" features not included at all, rather than listed as "no". They are greyed and struck out at the moment as a kludge for markup; that style makes the boxes the same height.